astroquery/ned/__init__.py

```diff
@@ -8,6 +8,8 @@
 
 .. topic:: Revision History
```

Comment:
Unrelated to the current PR:
 think these metadata sections should be cleaned up from all over, as git contains all the info and things move on from their original author etc. 

@keflavich @ceb8 - what do you think?